vnext/ReactUWP/ReactUWP.vcxproj

```diff
@@ -50,7 +50,8 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
-    <UnmergedWinmdDirectory>$([MSBuild]::NormalizePath($(IntDir)UnmergedWinMD))</UnmergedWinmdDirectory>
+    <!-- For relative paths, make it relative to MSBuildProjectDirectory not CurrentWorkingDirectory -->
+    <UnmergedWinmdDirectory>$([MSBuild]::NormalizeDirectory( $([System.IO.Path]::Combine( $(MSBuildProjectDirectory), $(IntDir)UnmergedWinMD )) ))</UnmergedWinmdDirectory>
```

Comment:
This looks more compact, but still not clear.
Where would the unmerged MD dir be located?
Being a derived artifact (though intermediate), it should go within the `IntDir` hierarchy.

Reply:
IntDir is a relative path, with the intent that it is relative to the project directory. However, NormalizeDirectory and NormalizePath resolve their inputs to an absolute path based on current working directory. Often times, that means C:\windows\system32.

For example:
IntDir = \build\x64\
NormalizedDir = c:\windows\system32\build\x64

Clearly that's no good. By using Path.Combine, I ensure the path is first resolved to absolute based on the project directory.

IntDir = \build\x64\
MSBuildProjectDir = c:\react-native-windows\vnext\reactcommon\
CombinedDir = c:\react-native-windows\vnext\reactcommon\build\x64
NormalizedDir =  c:\react-native-windows\vnext\reactcommon\build\x64\

Path.Combine is also smart -- if the second arg is already absolute, it just uses it as is.
Path.Combine("c:\foo", "d:\bar") == "d:\bar"

Reply:
I'm aware by default `IntDir` is something like `<thisdir>\obj\$(Platform)`.

In this case, we made `IntDir` NOT relative to the current directory.
See `vnext\Directory.Build.props`.

That said, as long as this doesn't mess with the intermediate artifact directory structure (`<root>\vnext\build\$(Platform)\$(Configuration)\$(ProjectName)`), this change should be fine.